[00;33m 	'cuda/11.3' does not supports the {CUDA_MPI}. [0m
[00;34m 	{CUDA_MPI} is only supported in cuda 11.4 version. [0m
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Traceback (most recent call last):
  File "main_gpt2_with_health_checker.py", line 467, in <module>
    trainer = Trainer(world_size, rank, bucket_size, count, adaptive_shard_ratio, health_check_scheduler_thread, health_check_main_proc, health_check_thread_ready, trial_info, thread) 
  File "main_gpt2_with_health_checker.py", line 291, in __init__
    schedule(self.adaptive_sdp_modules, max_param_num, \
TypeError: schedule() got an unexpected keyword argument 'net_bench_file_name'
srun: error: gpu10: task 0: Exited with exit code 1
slurmstepd: error: *** JOB 154347 ON gpu10 CANCELLED AT 2022-12-04T18:54:15 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 154347.0 ON gpu10 CANCELLED AT 2022-12-04T18:54:15 DUE TO TIME LIMIT ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
